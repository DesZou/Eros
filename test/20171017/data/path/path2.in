5 8
760519750 864353254 419787997 958957347 573108302 
1 2 229109529
2 5 471138788
4 2 344574664
1 3 405116854
2 3 389124415
4 1 67295934
5 4 644526888
1 5 112012517
