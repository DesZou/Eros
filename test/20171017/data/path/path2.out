0 198031566913557366 308099368524866500 64533930332527098 85187731425710750 
198031566913557366 0 336340954316096410 219706266102259563 198031566913557366 
308099368524866500 336340954316096410 0 373153716661327005 308099368524866500 
64533930332527098 219706266102259563 373153716661327005 0 107415226133112399 
85187731425710750 198031566913557366 308099368524866500 107415226133112399 0 
